packages/fftw3/fftw3.0.8.1/opam

```diff
@@ -0,0 +1,39 @@
+opam-version: "1.2"
+authors: ["Christophe Troestler <Christophe.Troestler@umons.ac.be>"]
+maintainer: "Christophe.Troestler@umons.ac.be"
+license: "LGPL-2.1 with OCaml linking exception"
+homepage: "https://github.com/Chris00/fftw-ocaml"
+dev-repo: "https://github.com/Chris00/fftw-ocaml.git"
+bug-reports: "https://github.com/Chris00/fftw-ocaml/issues"
+doc: "https://Chris00.github.io/fftw-ocaml/doc"
+tags: ["FFT"]
+build: [
+  [ "jbuilder" "subst" ] {pinned}
+  [ "jbuilder" "build" "-p" name "-j" jobs ] { os != "darwin" }
+  [ "env" "FFTW3_CFLAGS=-I /usr/local/include -L /usr/local/lib -I /usr/local/include -L /usr/local/lib"
+    "jbuilder" "build" "-p" name "-j" jobs ] { os = "darwin" }
+]
+build-test: [[ "jbuilder" "runtest" "-p" name "-j" jobs ]]
+depends: [
+  "jbuilder" {build}
+  "cppo"     {build}
+  "configurator" {build}
+  "base"     {build}
+  "stdio"    {build}
+  "archimedes" {test}
+  "lacaml" {test}
+]
+available: [ ocaml-version >= "4.03.0" ]
+depexts: [
```

Comment:
Also, it would be wise to move the depexts to an external meta `conf-fftw` package I think. This would avoid redundancies, but it can be done later.

Reply:
What redundancies?  IMHO, any package that wants to use FFTW should use this one (or contribute to it if it is not enough) and not depend again on the C code.  Or did I misunderstand your point?

Reply:
I'm just saying that gaving a single meta package that lists the external dependency for every systems is slightly better than having several versions of this package repeating this list (it might also need some adjustments and it's much better to have one meta package to handle everything than several that might be forgotten.